release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.8.0.0.md

```diff
@@ -0,0 +1,19 @@
+## Version 2.8.0.0 2023-06-06
+
+Compatible with OpenSearch 2.8.0
+
+### Features
+* Feature: Add refresh index operation to UI  ([#762](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/762))
+* Feature: Add clear cache operation to UI  ([#728](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/728))
+* Feature: Add flush index operation to UI  ([#757,719,752,755](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/757))
```

Comment:
I think if there are more than one PRs for one feature, we can add them like below to make each PR clickable:
([#757](https://github.com/opensearch-project/notifications/pull/757),[#719](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/719))